packages/editor/src/store/test/selectors.js

```diff
@@ -1306,6 +1267,15 @@ describe( 'selectors', () => {
 
 	describe( 'isEditedPostAutosaveable', () => {
 		it( 'should return false if the post is not saveable', () => {
+			const isEditedPostAutosaveable = isEditedPostAutosaveableRegistrySelector( () => ( {
```

Comment:
Was there some prior art for this convention? It seems reasonable, albeit a bit awkward; the nature of these registry selectors I'd assume.

Reply:
I couldn't see any. It doesn't seem so bad, but yeah, just a bit awkward.